export const alertError = () =>
  alert('Something went wrong, please try again!');
